content/docs/user-guide/experiment-management/checkpoints.md

```diff
@@ -0,0 +1,409 @@
+new set of checkpoints under a new experiment branch.
+
+```dvc
+┏━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━┳━━━━━━┳━━━━━━━━━┳━━━━━━━━┳━━━━━━━━┳━━━━━━━━┳━━━━━━━━━━━━━━┓
+┃ Experiment            ┃ Created  ┃ step ┃    loss ┃    acc ┃ seed   ┃ lr     ┃ weight_decay ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━╇━━━━━━╇━━━━━━━━━╇━━━━━━━━╇━━━━━━━━╇━━━━━━━━╇━━━━━━━━━━━━━━┩
+│ workspace             │ -        │    4 │ 0.41666 │ 0.8777 │ 473987 │ 0.0001 │ 0            │
+│ main                  │ 04:49 PM │    - │       - │      - │ 473987 │ 0.001  │ 0            │
+│ │ ╓ exp-a75bb         │ 05:18 PM │    4 │ 0.41666 │ 0.8777 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ b8a4ceb           │ 05:18 PM │    3 │ 0.50835 │ 0.8538 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ 7b652b1           │ 05:17 PM │    2 │ 0.72421 │ 0.8284 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ cf0168f           │ 05:17 PM │    1 │  1.2537 │ 0.7738 │ 473987 │ 0.0001 │ 0            │
+│ ├─╨ 8c7f40f           │ 05:17 PM │    0 │  1.9428 │ 0.5715 │ 473987 │ 0.0001 │ 0            │
+│ │ ╓ exp-7fafd         │ 05:10 PM │    5 │ 0.36601 │ 0.8943 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ 9ac9e6a           │ 05:09 PM │    4 │ 0.41666 │ 0.8777 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ b869efe           │ 05:09 PM │    3 │ 0.50835 │ 0.8538 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ 422395e           │ 05:09 PM │    2 │ 0.72421 │ 0.8284 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ 44e44a9           │ 05:09 PM │    1 │  1.2537 │ 0.7738 │ 473987 │ 0.0001 │ 0            │
+│ ├─╨ c6e11b4           │ 05:09 PM │    0 │  1.9428 │ 0.5715 │ 473987 │ 0.0001 │ 0            │
+```
+
+We can also remove all of the experiments we don't promote to our Git workspace
+with the following command:
+
+```bash
+dvc exp gc --workspace --force
+```
+
+## Adding checkpoints to Git
+
+When you terminate training, you'll see a few commands in the terminal that will
+allow you to add these changes to Git.
+
+```dvc
+To track the changes with git, run:
+
+        git add dvclive.json dvc.yaml .gitignore train.py dvc.lock
+
+Reproduced experiment(s): exp-263da
+Experiment results have been applied to your workspace.
+
+To promote an experiment to a Git branch run:
+
+        dvc exp branch <exp>
+```
+
+You can run the following command to save your experiments to the Git history.
+
+```bash
+git add dvclive.json dvc.yaml .gitignore train.py dvc.lock
+```
+
+You can take a look at what will be commited to your Git history by running:
+
+```bash
+git status
+```
+
+You should see something similar to this in your terminal.
+
+```git
+Changes to be committed:
+  (use "git restore --staged <file>..." to unstage)
+        new file:   .gitignore
+        new file:   dvc.lock
+        new file:   dvclive.json
+
+Untracked files:
+  (use "git add <file>..." to include in what will be committed)
+        data/
+        dvclive.html
+        dvclive/
+        model.pt
+        plots.html
+        predictions.json
+```
```

Comment:
It looks like a commit is missing: 
```
git commit -m 'my experiment'
```

and, probably a last "motivational" sentence about checkpoint 😄